events/isc-fall-2017.md

```diff
@@ -24,6 +24,22 @@ Some examples of possible content:
 
 Presenters of accepted papers will be notified July 31st. Note that all presentations given at the Fall Summit, unless otherwise designated by the presenter, will be covered by [the Chatham House Rule](https://www.chathamhouse.org/about/chatham-house-rule).
 
+### What is InnerSource?
+InnerSource takes the lessons learned from developing open source software and applies them to the way companies develop software internally. As developers have become accustomed to working on world class open source software, there is a strong desire to bring those practices back inside the firewall and apply them to software that companies may be reluctant to release. For companies building mostly closed source software, InnerSource can be a great tool to help break down silos, encourage internal collaboration, accelerate new engineer on-boarding, and identify opportunities to contribute software back to the open source world.
+ 
+### What is the InnerSource Commons?
+ InnerSourceCommons.org (aka ISC) utilizes open source methods to provide organizations pursuing inner sourcing a forum for discussing and improving the practice of inner source through the sharing of experiences (under the [Chatham House Rule](https://www.chathamhouse.org/about/chatham-house-rule)), creation and review of inner source patterns, and the open exchange of ideas.
```

Comment:
Should we include: 
InnerSource Commons is a consortium of representatives from over 60 companies and organizations...